lib/chef/resource/aws_security_group.rb

```diff
@@ -1,8 +1,15 @@
 require 'chef/provisioning/aws_driver/aws_resource_with_entry'
 require 'chef/resource/aws_vpc'
 
-class Chef::Resource::AwsSecurityGroup < Chef::Provisioning::AWSDriver::AWSResourceWithEntry
-  aws_sdk_type AWS::EC2::SecurityGroup
+class MultipleSecurityGroupError < RuntimeError
```

Comment:
TODO: extract this into `errors.rb` and have them descend from one base chef-provisioning-aws error.